pagopa/io-web-profile
# [IOPID-840] : Adding Mixpanel Track Event Logout-Flow

## Short Description
Adding Mixpanel Track Event for the **Logout-Flow.**

## How to Test
Run the project and check the console for the tracked event (mixpanel mocked), as specified in Figma, for the logout flow.
